src/main/java/github/grace000/erversay/RequestParser.java

```diff
@@ -0,0 +1,53 @@
+package github.grace000.erversay;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class RequestParser {
+    public String handleEmptyRequest(String requestString) {
+        if (requestString == null || requestString.isEmpty()) {
+            System.out.println("REQUEST CANNOT BE PARSED");
+            requestString = "GET /NOT_FOUND HTTP/1.1";
+        }
+        return requestString;
```

Comment:
This definitely works, but I wonder if you can't just handle it at the router level to automatically call `getResponseForNotFound()`?

Reply:
I'll handle this at the route level when I refactor the router.